### ISSUE TITLE
Add changeset plugin

### ISSUE THREAD

**Issue by hipstersmoothie** [intuit/auto#1807]
This plugin should:

- turn off auto changelog features
- get version bumps from changeset files, apply as labels
- be able to pair with `npm` plugin
- use changeset for GitHub release

**Comment by laughedelic** [comment]
Could you clarify what are the "changeset files"?

**Comment by vincentbriglia** [comment]
@laughedelic I assume the wonderful library from atlassian https://github.com/changesets ... auto trumps it though, auto is more automated, changesets is manual

**Comment by hipstersmoothie** [comment]
Yeah this issue was more to explore what combining the two might feel like. I'm not sure it would even work that well.

The main thing I like about changesets is the ability to comment on them like normal changed files so they can go through review.

---

> auto trumps it though, auto is more automated, changesets is manual

These are very kind words 😄  Maybe I should add a testimonials section or even a "companies using auto" section to the marketing page
